### PR TITLE
feat: use Logster for better request logging

### PR DESCRIPTION
**Pull request by lemald**
To aid in [⚙️Benchmark tabs / presets API performance](https://app.asana.com/0/1200180014510248/1201651613270218/f)

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -76,7 +76,8 @@ defmodule Skate.MixProject do
       {:postgrex, "~> 0.15"},
       {:ex_aws_rds, "~> 2.0.2"},
       {:fast_local_datetime, "~> 1.0"},
-      {:ex_machina, "~> 2.7.0", only: :test}
+      {:ex_machina, "~> 2.7.0", only: :test},
+      {:logster, "~> 1.0"}
```
Comment (arkadyan) [inline mix.exs:80]: Not related to this line specifically, but…

`logger` isn't defined here in the `deps` list, but it is still above in the list of `extra_applications`. Is it still required to be there for other libraries or Phoenix or something?

File: config/prod.exs [hunk 2]
```diff
@@ -53,6 +53,9 @@ config :logger, :console,
   format: "$time $metadata[$level] node=$node $message\n",
   metadata: [:request_id]
 
+# "code" is the secret value returned by AWS to /auth/cognito/callback
```
Comment (arkadyan) [inline config/prod.exs:56]: 👍

**Review by arkadyan — APPROVED** [review]
Small question just to check whether Logger still needs to be running, but otherwise looks good. 👍

File: mix.exs (hunk 1 above)
Comment (lemald) [reply mix.exs:80]: My understanding is that Logster is just a plug that calls Logger with a nicer format than the default, so we still want that running.

Comment (lemald) [reply mix.exs:80]: It looks like the API [also leaves the `:logger` process running](https://github.com/mbta/api/blob/master/apps/api_web/mix.exs#L28), for a point of comparison, so I feel pretty good about merging this as-is.

**Pull request merged**